src/translations/en.json

```diff
@@ -3367,5 +3367,47 @@
         }
       }
     }
+  },
+  "supervisor": {
+    "addon": {
+      "panel": {
+        "configuration": "Configuration",
+        "documentation": "Documentation",
+        "info": "Info",
+        "log": "Log"
+      }
+    },
+    "common": {
+      "cancel": "Cancel",
+      "newest_version": "Newest Version",
+      "release_notes": "Release notes",
+      "update": "Update",
+      "version": "Version"
+    },
+    "confirm": {
+      "update": {
+        "title": "Update ${name}",
+        "text": "Are you sure you want to update {name} to version {version}?"
+      }
+    },
+    "dashboard": {
+      "addon_new_version": "New version available",
+      "addon_running": "Add-on is running",
+      "addon_stopped": "Add-on is stopped",
+      "addons": "Add-ons",
+      "no_addons": "You don't have any add-ons installed yet. Head over to the add-on store to get started!",
+      "update_available": "Update pending {icon}",
+      "updates_available": "Updates pending {icon}"
```

Comment:
```suggestion
      "update_available": "{count, plural,\n  one {Update}\n  other {{count} Updates}\n} pending {icon}",
```

Reply:
You can use plural syntax in translations, as not all languages have just one/other :-)

Reply:
That looks like magic

Reply:
<https://formatjs.io/docs/intl-messageformat#common-usage-example>